WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -388,6 +388,25 @@ class PagesFragment : Fragment() {
                 )
             }
         })
+
+        viewModel.publishAction.observe(this, Observer {
+            it?.let {
+                UploadUtils.publishPost(activity, it.post, it.site, dispatcher)
```

Comment:
Hi Jirka! This is probably a nitpick but I was thinking we could add this function to the `UploadUtilsWrapper` since it's being used in this class. A function above this that wasn't in this diff is also using `UploadUtils` as well. Wdyt?

Reply:
Good call, thanks! Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/11347/commits/5f5e4a593b902f08ac081c2239a7eb9efbda04d6

Reply:
Awesome thanks!